source/expressions.tex

```diff
@@ -3345,7 +3345,7 @@
 If the function is an implicit object member
 function,
 the object expression of the class member access shall be a glvalue and
-the \keyword{this} parameter of the function\iref{expr.prim.this}
+the implicit object parameter of the function\iref{over.match.funcs}
```

Comment:
Is this right? https://eel.is/c++draft/over.match.funcs#general-4 says that the implicit object parameter is of type "reference to X" -- how can you initialize that with a pointer to the obeject of the call?

Reply:
It seems that we can say the implicit object parameter is initialized with the aforementioned glvalue.

Reply:
@jensmaurer Any thoughts on this new version?

Reply:
Yeah, I think that's a mild improvement.  We still seem to be lacking precise wording connecting `this` to the value of the implicit object parameter.

Reply:
OK, let's find wording that we're enthusiastically happy about first if we're already going to change this.

Reply:
That's a core issue, though.

Reply:
Would you still find this a valuable intermediate step to simplify the core issue, or would you rather do it as part of the core issue after all?

Reply:
Yes, as I said, it's a mild improvement over the status quo, because a "this parameter" isn't a thing.

Reply:
Yes, sure, but I was specifically wondering whether it's a helpful change to simplify the CWG issue. If, say, the issue would entirely replace this text with something totally different, then it might not be worth making this small change now.

Reply:
Yes, I think this update is a step in the right direction. A future core issue will add to that and also amend other text, but getting rid of "this parameter" is definitely the right thing to do.

Reply:
Alright, thanks, let's apply it then!